src/Appwrite/Platform/Workers/Deletes.php

```diff
@@ -450,6 +450,23 @@ private function deleteProject(Database $dbForConsole, callable $getProjectDB, c
             Query::equal('projectInternalId', [$projectInternalId])
         ], $dbForConsole);
 
+        // Delete VCS Installations
+        $this->deleteByGroup('installations', [
+            Query::equal('projectInternalId', [$projectInternalId])
+        ], $dbForConsole);
+
+        // Delete VCS Repositories and Comments
+        $this->deleteByGroup('repositories', [
+            Query::equal('installationInternalId', [$projectInternalId]),
```

Comment:
this should not be `installationInternalId` it should be the `projectInternalId`